centerGraphic.sh

```diff
@@ -1,11 +1,23 @@
 #!/bin/sh
 set -euo pipefail
 
-DEFAULT_COLS=$(stty size 2>/dev/null | awk {'print $2'} || echo "1")
+LANG="C.UTF-8"
```

Comment:
I would like to maintain this line next to the MAX_LENGTH line because they are closely related

Reply:
As an expert in dburrito's PR I would like to give @dgonsudo my 10s.


ddrunk did it again. This is great.